### PR TITLE
feat(cellguide): add script to populate rdev with latest cellguide data

### PR DESCRIPTION
**Pull request by atarashansky**
## Reason for Change

- Added a script to make it easy to populate new rdev stacks with the latest cellguide data.
- Run `bash scripts/populate_rdev_with_cellguide_data.sh --help` for details on how to use this command.

### CODE REVIEW

File: scripts/populate_rdev_with_cellguide_data.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    echo "Usage: $0 [-s|--src_deployment <src_deployment>] <stack_name>"
+    echo "Make sure AWS_PROFILE is set to single-cell-dev prior to running this command."
+    echo ""
+    echo "Populates the specified rdev stack with data."
+    echo ""
+    echo "Options:"
+    echo "  -s, --src_deployment    The source deployment to use. Can be either 'staging' or 'dev'. Default is 'staging'."
+    echo ""
+    echo "Arguments:"
+    echo "  stack_name              The name of the rdev stack to be populated."
```
Comment (tihuan) [inline scripts/populate_rdev_with_cellguide_data.sh:15]: Seems unusually long spaces. Is that intentional?

File: scripts/populate_rdev_with_cellguide_data.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+SRC_DEPLOYMENT="staging"
+
+function usage {
+    echo "Usage: $0 [-s|--src_deployment <src_deployment>] <stack_name>"
+    echo "Make sure AWS_PROFILE is set to single-cell-dev prior to running this command."
+    echo ""
+    echo "Populates the specified rdev stack with data."
+    echo ""
+    echo "Options:"
+    echo "  -s, --src_deployment    The source deployment to use. Can be either 'staging' or 'dev'. Default is 'staging'."
```
Comment (tihuan) [inline scripts/populate_rdev_with_cellguide_data.sh:12]: Same here - long spaces

**Review by tihuan — APPROVED** [review]
Just some minor comments otherwise LGTM!

File: scripts/populate_rdev_with_cellguide_data.sh (hunk 1 above)
Comment (atarashansky) [reply scripts/populate_rdev_with_cellguide_data.sh:15]: Yeah, it's just some formatting
image

Comment (tihuan) [reply scripts/populate_rdev_with_cellguide_data.sh:15]: ah makes sense now thank you!

**Pull request merged**